packages/prettier-plugin-java/src/printers/classes.js

```diff
@@ -50,23 +50,32 @@ class ClassesPrettierVisitor {
   }
 
   typeParameters(ctx) {
-    return "TBD";
+    const typeParameterList = this.visit(ctx.typeParameterList);
```

Comment:
Maybe call it "optionalTypeParameterList" so it is easier to understand the use of "reject"?